.github/workflows/tasklist-migrate-elasticsearch-data.yml

```diff
@@ -64,12 +64,12 @@ jobs:
       # Build: maven artifacts
       - name: Build backend
         run: |
-          mvn -f tasklist -B -DskipTests -P skipFrontendBuild clean install
+          mvn -B -DskipTests -DskipChecks -T1C -P skipFrontendBuild clean install
 
       # Tests: run migration tests
       - name: Run migration tests
         run: |
-          mvn -B -f tasklist/qa/migration-tests -DskipTests=false verify
+          mvn -B -pl tasklist/qa/migration-tests -DskipTests -DskipChecks verify
```

Comment:
❓ Previously we didn't skip the tests, now we do. Was this intended?

Reply:
Good point, this is a mistake